Configurations/15-android.conf

```diff
@@ -1,7 +1,7 @@
 #### Android...
 #
-# See NOTES.ANDROID for details. [But don't miss platform-specific
-# comments below.]
+# See NOTES.ANDROID for details. But don't miss platform-specific
+# comments below...
```

Comment:
Make it one sentence and change "but" to "and"...  i.e. "... details, and don't miss ..."

Reply:
Done.